src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2082,7 +2082,7 @@ SELECT histogram(duration, 10, 20) FROM PageView SINCE 1 week ago
     * Use [`TIMESERIES`](#sel-timeseries) to generate a line chart with rates mapped over time.
     * Omit [`TIMESERIES`](#sel-timeseries) to generate a billboard showing a single rate value averaged over time.
 
-      Here's a basic query that will generate a line chart showing the rate of throughput for <InlinePopover type="apm" /> transactions per 10 minutes over the past 6 hours:
+    Here's a basic query that will generate a line chart showing the rate of throughput for <InlinePopover type="apm" /> transactions per 10 minutes over the past 6 hours:
```

Comment:
maybe "that will generate" should be "that generates" to make it more direct? Super duper small, just a thought.